### PR TITLE
fix(GlobalStatus): error state not applied on prop change

### PR DESCRIPTION
**Pull request by joakbjerk**

### CODE REVIEW

**Review by langz — APPROVED** [review]
🕵️‍♂️

File: packages/dnb-eufemia/src/components/global-status/GlobalStatus.js [hunk 1]
```diff
@@ -352,7 +352,7 @@ export default class GlobalStatus extends React.PureComponent {
       const globalStatus = extendPropsWithContextInClassComponent(
         this.props,
         GlobalStatus.defaultProps,
-        this.state.globalStatus
+        this.context.globalStatus
```
[the remark this answers is not in the document]
Comment (joakbjerk) [reply packages/dnb-eufemia/src/components/global-status/GlobalStatus.js:355]: 🙏

**Review by tujoworker — COMMENTED** [review]
Nice tests 👌

**Pull request merged**